lib/workers/repository/process/index.spec.ts

```diff
@@ -32,5 +33,57 @@ describe('workers/repository/process/index', () => {
       // FIXME: explicit assert condition
       expect(res).toMatchSnapshot();
     });
+
+    it('reads config from default branch if useBaseBranchConfig not specified', async () => {
+      git.branchExists.mockReturnValue(true);
+      platform.getJsonFile = jest.fn().mockResolvedValue({});
```

Comment:
```suggestion
      platform.getJsonFile.mockResolvedValueOnce({});
```

Platform is already mocked